browser/main/Detail/ToggleDirectionButton.js

```diff
@@ -0,0 +1,26 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import CSSModules from 'browser/lib/CSSModules'
+import styles from './ToggleDirectionButton.styl'
+import i18n from 'browser/lib/i18n'
+
+const ToggleDirectionButton = ({
+  onClick, editorDirection
+}) => (
+  <div styleName='control-toggleModeButton'>
+    <div styleName={editorDirection ? 'active' : undefined} onClick={() => onClick()}>
```

Comment:
`editorDirection` is quite hard to understand, can you assign it to a more meaningful variable name such as `isRTL` or something?